MDANSE/Src/MDANSE/Framework/QVectors/CircularQVectors.py

```diff
@@ -18,16 +18,84 @@
 import collections
 
 import numpy as np
+import numpy.typing as npt
+from scipy.spatial.transform import Rotation
 
-from MDANSE.Framework.QVectors.IQVectors import IQVectors
-from MDANSE.Mathematics.Geometry import random_points_on_circle
+from MDANSE.Framework.QVectors.IQVectors import IQVectors, truncated_normal_distribution
+
+
+def circle_rotation_matrix(
+    target_circle_axis: npt.NDArray[float],
+) -> npt.NDArray[float]:
+    """Return rotation matrix that transforms 001 vector into target vector.
+
+    Parameters
+    ----------
+    target_circle_axis : npt.NDArray[float]
+        A 3D vector defining the axis of a circle.
+
+    Returns
+    -------
+    npt.NDArray[float]
+        a 3x3 rotation matrix
+    """
+    try:
+        rotation, _ = Rotation.align_vectors(target_circle_axis, np.array([0, 0, 1]))
+    except ValueError:
+        return None
+    else:
+        return rotation.as_matrix()
```

Comment:
No need for else after return


```suggestion
    return rotation.as_matrix()
```